internal/tokencache/access_token_cache_item_test.go

```diff
@@ -48,8 +50,8 @@ func TestCreateAccessTokenCacheItem(t *testing.T) {
 		"user.read",
 		"access",
 	)
-	if !reflect.DeepEqual(extExpiresOn, *actualAt.ExtendedExpiresOnUnixTimestamp) {
-		t.Errorf("Actual ext expires on %s differs from expected ext expires on %s", *actualAt.ExtendedExpiresOnUnixTimestamp, extExpiresOn)
+	if !reflect.DeepEqual(extExpiresOn, actualAt.ExtendedExpiresOnUnixTimestamp) {
```

Comment:
I see on line 106, we moved from using `reflect.DeepEqual` to a simple comparison. 
Can we do this here as well ?

Reply:
Removed use of reflect completely from this file.